ts-fold.el

```diff
@@ -82,7 +81,7 @@
     (php-mode        . ,(ts-fold-parsers-php))
     (python-mode     . ,(ts-fold-parsers-python))
     (rjsx-mode       . ,(ts-fold-parsers-javascript))
-    (ruby-mode       . ,(ts-fold-parsers-ruby))
+    (ruby-ts-mode    . ,(ts-fold-parsers-ruby))
```

Comment:
With emacs 29, the treesitter enabled modes are actually seperate from the built in ones, so this alist entry had to change